Sources/Runner/Commands/Edit.swift

```diff
@@ -44,15 +44,12 @@ func editDanger(logger: Logger) throws {
 
     let importsFinder = ImportsFinder()
     let importedFiles = importsFinder.findImports(inString: dangerfileContent)
+        .map { importsFinder.resolveImportPath($0, relativeTo: dangerfilePath) }
 
     let scriptManager = try getScriptManager(logger)
     let script = try scriptManager.script(atPath: dangerfilePath)
 
-    let configPath = NSTemporaryDirectory() + "config.xcconfig"
-
-    try createConfig(atPath: configPath, libPath: absoluteLibPath, libsImport: libsImport)
-
-    try script.setupForEdit(importedFiles: importedFiles, configPath: configPath)
```

Comment:
No flags can be set on Package.swift, and needs `Danger` package as a default dependency instead.
https://github.com/danger/swift/pull/566/files?w=1#diff-b040c9d7cf1d8a4d0275190fb078df8712464dcb18a89b281f9a93145610068dR16